src/3d.js

```diff
@@ -706,6 +706,10 @@ var getMeshFromCache = getCache(function (drawing, pal, type) {
 	mesh.makeGeometryUnique();
 	mesh.isVisible = false;
 	mesh.material = getMaterial(drawing, pal);
+	// enable vertical tiling for towers
+	if (type.startsWith('tower')) {
+		mesh.material.diffuseTexture.wrapV = BABYLON.Texture.WRAP_ADDRESSMODE;
```

Comment:
note that this modifies the assigned texture directly, as opposed to modifying a property of the mesh; in the common use case this will work as expected since textures are unique to each mesh, but if you were to use this along with something like the drawing replacement hack, this would result in textures shared between towers and other assets to wrap on the other assets as well